credit-conveyor/Conveyor/src/main/java/com/deals/customEnum/Passport.java

```diff
@@ -0,0 +1,9 @@
+package com.deals.customEnum;
+
+public enum Passport {
```

Comment:
Passport isn't enum, it is plain java object.
You have to set series, number, etc. but in enum you can't do it